.ci/pipeline/dr_weekly.yaml

```diff
@@ -0,0 +1,89 @@
+---
+job: LIBXLIO-dr-weekly-launcher
+failFast: false
+timeout_minutes: 120
+
+runs_on_agents:
+  - {nodeLabel: 'master', category: 'base'}
+
+env:
+  MAIL_FROM: jenkins@nvidia.com
+
+pipeline_start:
+  shell: action
+  module: groovy
+  run: |
+    if (!params.MAIL_TO.isEmpty()) {
+      currentBuild.displayName += "_weekly-on-${sha1}"
+      mail from: "${MAIL_FROM}",
+        mimeType: 'text/html',
+        to: "${MAIL_TO}",
+        subject: 'LibXLIO weekly DR build has been started',
+        body: """
+          <p><b>Branch:</b> ${sha1}</p>
+          <p><b>Build url:</b> <a href=${currentBuild.absoluteUrl}>link</a></p>"""
+    }
+
+steps:
+  - name: Determine release_tag
+    agentSelector:
+      - "{category: 'base'}"
+    shell: action
+    module: groovy
+    run: |
+      if (params.release_tag.isEmpty()) {
```

Comment:
So what if we do have release_tag parameter defined you are not setting RELEASE_TAG env var we will not be able to run (see line 55).
We have 2 choices have this parameter and make sure it is used when defined or not have it and remove all related logic

Reply:
- Agree, added another handling for the case when release_tag is defined.